### ISSUE TITLE
A lot of these dashboards fail to import.

### ISSUE THREAD

**Issue by ericsheris-rumble** [openobserve/dashboards#2]
I have not tried all of them, but I was unable to get either of the dashboards in the OpenObserve folder to import. 
These also failed
Kubernetes(openobserve-collector)/Kubernetes  _ Namespace (Pods).dashboard.json
Kubernetes(openobserve-collector)/Kubernetes  _ Namespaces.dashboard.json
Kubernetes(openobserve-collector)/Kubernetes  _ Namespace (Pod).dashboard.json
all of the dashboards in Kubernetes(kube-prom-stack)
these all failed with either something like 
```
Kubernetes _ Namespace (Pods).dashboard.json : AxiosError: Request failed with status code 403
```
or just pop up at the bottom to enter a json object to import when trying to import from url or json directly

**Comment by prabhatsharma** [comment]
What version of OpenObserve are you running?

**Comment by ericsheris-rumble** [comment]
I just finished following the OpenObserve HA setup via helm chart. Values file says v0.9.1?

**Comment by ericsheris-rumble** [comment]
I was able to get some dashboards to import successfully such as the Traces dashboard and the Kubernetes nodes and events dashboards, though the events dashboard doesn't display any data atm. 

I had installed this a couple weeks back and had similar issues, but we are re-evaluating the product.

**Comment by prabhatsharma** [comment]
Just updated the dashboards. Can you try again? Dashboard structure has been evolving to add more features, and dashboards in this repo have gotten old. New ones should work.

**Comment by ericsheris-rumble** [comment]
so i tried Kubernetes _ Namespace (Pods).dashboard.json which says it was updated 5 min ago. https://github.com/openobserve/dashboards/blob/main/Kubernetes(openobserve-collector)/Kubernetes%20%20_%20Namespace%20(Pods).dashboard.json for ref
I tried importing it via downloading the json file (Axioserror: Request failed with status code 403)
![image](https://github.com/openobserve/dashboards/assets/73132669/d0e3b901-74ab-4973-96ae-48ac08988dae)

Import from url, tried to use https://raw.githubusercontent.com/openobserve/dashboards/main/Kubernetes(openobserve-collector)/Kubernetes%20%20_%20Namespace%20(Pods).dashboard.json. This just pops up and says "Please enter a url for import" but i have a url there.
tried just pasting the json from the directly above raw.githubusercontent.com link into the "Import Dashboard from JSON" and it acts like i did not paste a big json blob into it, but i can clearly see it
![image](https://github.com/openobserve/dashboards/assets/73132669/7a8bcc81-9165-4011-bdac-8d4406d72aaa)

**Comment by prabhatsharma** [comment]
I created this short video on how to import dashboards. See if this helps. https://www.youtube.com/watch?v=aPZlj2fUcwo

**Comment by ericsheris-rumble** [comment]
i watched your video, i did the same thing you did, it did not import. I cloned the whole repo and tried again, it still does not import. Here's my version info
![image](https://github.com/openobserve/dashboards/assets/73132669/32fc0537-720c-40d4-bf75-ffa7d3f9d1f5)
and here's a gif of the failure happening
![openobserve-dashboardimportfailure](https://github.com/openobserve/dashboards/assets/73132669/6aa6bf7a-6c64-4a6f-8016-b9398fb13107)

**Comment by prabhatsharma** [comment]
@ktx-kirtan - Any thoughts? I have not been able to reproduce any failures mentioned by @ericsheris-rumble . I have tried these with 0.9.1 and 0.9.2-rc2 . Tried on windows and mac. All work for me.

**Comment by ktx-kirtan** [comment]
I tried and worked for me too. 

The video you shared shows error 403 (i.e forbidden) Looks like a permission issue. 

@ericsheris-rumble can you open Inspect, go to `Network` tab and share the request and response details? That would help us debug the issue.

**Comment by prabhatsharma** [comment]
@ericsheris-rumble Are you using enterprise version? Does your reverse proxy allow uploads? Is pasting JSON data in the UI also failing every time?

**Comment by ericsheris-rumble** [comment]
For reference, i'm running this in kubernetes using the HA setup. I didn't make any major modifications to the install. I think I figured it out though. I think its size related in the end. I had increased the nginx proxy buffer size for the openobserve ingress because there were some http 413 errors in the logs from the tracing i enabled. after increasing the buffer size, these dashboards also seem to import fine.
